src/init1.cc

```diff
@@ -360,7 +358,7 @@ static cptr r_info_flags6[] =
 	"TELE_AWAY",
 	"TELE_LEVEL",
 	"DARKNESS",
-	"TRAPS",
+	"TRAPS",	/* Unused */
```

Comment:
Just a little additional note: You may need to use "grep --text -v ^S:TRAPS [...]" on the Theme files. It seems grep thinks r_info.txt (Theme) is binary and chops off a huge portion of the file otherwise.